### PR TITLE
Added display name fields to account edit form.

### PR DESCRIPTION
**Pull request by sun**
Ticket: https://app.asana.com/0/633061025239778/815491131433230

### CODE REVIEW

File: assets/styles/woocommerce.css [hunk 1]
```diff
@@ -26,3 +26,15 @@
...
+
+/* Account: Place display name fields on one line. */
+form.edit-account > .display-name .form-row-first {
+  float: left;
+}
+form.edit-account > .display-name .form-row {
+  display: inline-block;
+  width: 48%;
+}
+form.edit-account > .display-name .form-row:nth-of-type(-n+1) {
```
Comment (fabianmarz) [inline assets/styles/woocommerce.css]: Why do you write it this complex? Why don't you use the `.form-row-first` class to set the margin on the first element? For mobile views the elements should also be one per line, don't you think?

File: src/Plugin.php [hunk 2]
```diff
@@ -512,8 +514,12 @@ public static function onOpenIdConnectLogin($user, $token_response, $id_token_cl
     if ($last_edit && $last_edit > $user_claims['profile_update_date']) {
       return;
     }
-    update_user_meta($user_id, 'first_name', $user_claims['firstname']);
-    update_user_meta($user_id, 'last_name', $user_claims['lastname']);
+    if (!empty($user_claims['display_firstname'])) {
+      update_user_meta($user_id, 'first_name', $user_claims['display_firstname']);
```
Comment (fabianmarz) [inline src/Plugin.php:518]: Shouldn't we update the values to be empty (remove the values) if neither nor is set to reflect the SSO values? Especially as those fields aren't mandatory fields.

Please see [this](https://app.asana.com/0/633061025239778/890333550594547/f) ticket.

File: src/WooCommerce.php [hunk 3]
```diff
@@ -1538,6 +1553,8 @@ public static function woocommerce_save_account_details_redirect($user_id) {
    */
   public static function woocommerce_edit_account_form() {
     $form = ob_get_clean();
+    preg_match_all('@^\s*<p [a-z_ "=-]+>\s+<label for="(?:account_first_name|account_last_name).+?</p>@sm', $form, $matches);
```
Comment (fabianmarz) [inline src/WooCommerce.php:1559]: Can we please add a comment to clarify what this regex do?

**Review by fabianmarz — CHANGES_REQUESTED** [review]

File: assets/styles/woocommerce.css (hunk 1 above)
Comment (sun) [reply assets/styles/woocommerce.css]: Thanks for catching this - was an obsolete effect of a previous approach.

**Comment by sun** [comment]
@fabianmarz Resolved all of your review issues, can you quickly review again?

**Review by fabianmarz — APPROVED** [review]

**Pull request merged**